client/lib/happychat/index.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Internal dependencies
+ */
+import {
+	HAPPYCHAT_GROUP_WPCOM,
+	HAPPYCHAT_GROUP_JPOP
+} from 'state/happychat/constants';
+import { isJetpackSite } from 'state/sites/selectors';
+
+/**
+ * Get happychat group for site id
+ * For now if the site is jetpack jpop will be set as the group otherwise WP.COM
+ * @param {object} state Current state
+ * @param {int} siteId The site id, if no siteId is present primary siteId will be used
+ * @returns {string} The happychat group for the selected site
+ */
+export const getGroupForSiteId = ( state, siteId ) => {
```

Comment:
Right, initially I thought it would make sense to separate them but for now there is no added value so I will just merge them into one.